src/angular/sdk-setup.ts

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+
+// @ts-expect-error - magicast is ESM and TS complains about that. It works though
+import { loadFile, writeFile } from 'magicast';
+
+import * as path from 'path';
+
+// @ts-expect-error - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import chalk from 'chalk';
+import { updateAppConfigMod } from './codemods/app-config';
+import type { SemVer } from 'semver';
+
+export function hasSentryContent(
```

Comment:
l (already suggested in #768): let's use the ast-utils version if possible

Reply:
Updated 👍